examples/tutorials/physics_benchmarking.py

```diff
@@ -0,0 +1,334 @@
+    sensor_specs.append(sensor_spec)
+
+    # agent configuration
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+
+    return habitat_sim.Configuration(backend_cfg, [agent_cfg])
+
+
+def simulate(sim, dt=1.0, get_frames=True, data=None):
+    # simulate dt seconds at 60Hz to the nearest fixed timestep
+    # print("Simulating " + str(dt) + " world seconds.")
+    observations = []
+    physics_step_times = []
+    graphics_render_times = []
+    collisions = []
+
+    start_time = sim.get_world_time()
+    while sim.get_world_time() < start_time + dt:
+        st = time.time()
+        sim.step_physics(1.0 / 60.0)
+        mt = time.time()
+        if get_frames:
+            observations.append(sim.get_sensor_observations())
+        et = time.time()
+        physics_step_times.append(mt - st)
+        graphics_render_times.append(et - mt)
+        collisions.append(sim.get_num_active_contact_points())
+
+    if "observations" in data:
+        data["observations"] += observations
+    if "physics_step_times" in data:
+        data["physics_step_times"] += physics_step_times
+    if "graphics_render_times" in data:
+        data["graphics_render_times"] += graphics_render_times
+    if "collisions" in data:
+        data["collisions"] += collisions
+
+
+# Define each test function & store in list
+
+
+def box_drop_test(
+    objects,
+    num_objects=30,
+    box_size=2,
+    object_speed=2,
+    post_throw_settling_time=5,
+):  # take in parameters here
+    """Drops a specified number of objects into a box and returns metrics including the time to simulate each frame, render each frame, and the number of collisions each frame. """
+    # start with box being bounding boxes
+
+    data = {
+        "observations": [],
+        "physics_step_times": [],
+        "graphics_render_times": [],
+        "collisions": [],
+    }
+
+    cfg = make_configuration()
+    with habitat_sim.Simulator(cfg) as sim:
+        sim.initialize_agent(
+            0,
+            habitat_sim.AgentState(
+                [5.45 * box_size, 2.4 * box_size, 1.2 * box_size],
+                np.quaternion(-0.83147, 0.2, -0.55557, -0.2),
+            ),
+        )
+
+        # get the physics object attributes manager
+        obj_templates_mgr = sim.get_object_template_manager()
+        # [/initialize]
+
+        # build box
+        cube_handle = obj_templates_mgr.get_template_handles("cube")[0]
+
+        box_parts = []
+        boxIDs = []
+        for _i in range(5):
+            box_parts += [obj_templates_mgr.get_template_by_handle(cube_handle)]
+        box_parts[0].scale = np.array([1.0, 0.1, 1.0]) * box_size
+        box_parts[1].scale = np.array([1.0, 0.6, 0.1]) * box_size
+        box_parts[2].scale = np.array([1.0, 0.6, 0.1]) * box_size
+        box_parts[3].scale = np.array([0.1, 0.6, 1.0]) * box_size
+        box_parts[4].scale = np.array([0.1, 0.6, 1.0]) * box_size
```

Comment:
You don't need duplicates here.

Reply:
If I don't use duplicates, then I can't use the loop below this and would have to manually register each box part. Is it worth it to remove duplicates and change the loop?